core/testMinimumPhpVersion.php

```diff
@@ -22,14 +22,21 @@
 // 3) composer.json (in two places)
 // 4) tests/PHPUnit/Integration/ReleaseCheckListTest.php
 $piwik_minimumPHPVersion = '7.2.5';
+$piwik_maximumPHPVersion = '8.0.0';
```

Comment:
Cheers @luggesexe would it need to be `8.0.0-dev`? so it also matches PHP 8 beta versions etc

Reply:
Yes this should work. I will add it.